docs/scenarios/library-authoring/pinning-libraries-to-rc2.md

```diff
@@ -0,0 +1,70 @@
+# How to manage package dependency versions for RC2
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers how to manage package versions when authoring a library for .NET Core.
+
+## Glossary
+
+**TFM** - Target Framework Moniker.  This is a string that represents the target you want your code to run on.  In this case of a library, this is `netstandard1.x`, where `1 <= x <= 5`.
+
+**Metapackage** - A NuGet package that represents a set of NuGet packages.  They're organized in a way that makes sense for general library and app development.
+
+## Pin your code to RC2
+
+When creating libraries with .NET Core RC2, it's important that you pin your library to RC2.  This means two things.  First, you should only depend on packages marked as final for .NET Core RC2.  Second, each package should come from the same build. This means every package should have this pattern: `<package_version>-rc2-<build_number>`.
+
+**Note: The build number at the end is not final at this time.  It will have a final number when RC2 ships.**
+
+**Examples of packages pinned to RC2**
+
+`"System.Collections":"4.0.11-rc2-24022"`
+
+`"NETStandard.Library":"1.5.0-rc2-24022"`
+
+`"Microsoft.NETCore.App":"1.0.0-rc2-24022"`
+
+**Examples of packages that are NOT pinned to RC2**
+
+`"System.Collections":"4.0.11-rc3-24021-00"`
+
+`"System.Net.Http":"4.1.0-rc2-24008"`
+
+`"System.Text.RegularExpressions":"4.0.10-rc3-24021-00"`
+
+### Why does this matter?
+
+We guarantee that if you pin your dependencies to RC2, those packages will all work with together.  There is no such guarantee if you use packages which aren't pinned to RC2.
+
+### Is there some big list to look at with version numbers and stuff?
+
+Yes, but depending on the nature of your library, you may not need to look through a big list.
+
+**Are you depending only on `NETStandard.Library`?**
+
+If so, all you need to do is pin your `NETStandard.Library` package to RC2.  Because this is a curated metapackage, its package closure is also pinned to RC2.
+
+**Are you depending only on `Microsoft.NETCore.App`?**
+
+If so, all you need to do is pin your `Microsoft.NETCore.App` package to RC2.  Because this is a curated metapackage, its package closure is also pinned to RC2.
+
+**Are you trimming your `NETStandard.Library` or `Microsoft.NETCore.App` metapackage dependencies?**
+
+If so, all you need to do is ensure that the metapackage you start with is pinned to RC2.  The individual packages you depend on after trimming are also pinned to RC2.
```

Comment:
What trimming are you referring to here? I assume you are implying manual trimming.